test/evergreen.yml

```diff
@@ -97,6 +97,19 @@ functions:
           set -o verbose
           echo '#undef HAVE_FTRUNCATE' >> wiredtiger_config.h
     - *make_wiredtiger
+  "compile wiredtiger sanitizer":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/build_posix"
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+          sh reconf
+          CC="clang -fsanitize=address" CFLAGS="-g -fno-omit-frame-pointer" \
```

Comment:
Default CFLAGS values are "-ggdb -fPIC". So, I think the CFLAGS should be set to "-ggdb -fPIC -fno-omit-frame-pointer" here and CC, CXX, PATH environment variable should point to mongodbtoolchains absolute path.

Reply:
The compilation is unique for this one: http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress-sanitizer/configure . These setting are invoking the address sanitizer and for that we are using clang rather than gcc. 

I will fix the `PATH` settings.

Reply:
You are actually correct about CFLAGS - will fix that as well.